source/fundamentals/collations.txt

```diff
@@ -4,5 +4,311 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to order your query
+or aggregation operation results by string values. A collation is a set of character
+ordering conventions that apply to a specific language and locale.
+
+Collations in MongoDB
+---------------------
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Certain languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following Canadian French words: 
+
+- cote
+- coté
+- côte
+- côté
+
+When using the default binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To specify a collation, create a ``Collation`` object. You must define the ``Locale`` field
+of the ``Collation`` object; all other fields are optional. For example, the following code
+snippet specifies a ``Collation`` object with the ``"en_US"`` locale collation:
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "en_US"}
+
+For a complete list of ``Collation`` object fields, visit the `Collation API documentation 
+<{+api+}/mongo/options#Collation>`__. To see all the supported locales and the
+default values for the ``Locale`` fields, visit :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+
+Usage
+-----
+
+For information on applying a collation to a collection, view, index, or operation, navigate
+to the following sections:
+
+- :ref:`golang-collection-collation`
+- :ref:`golang-index-collation`
+- :ref:`golang-op-collation`
```

Comment:
Issue: [three levels of headings](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure)

Suggestion: Remove the "Usage" heading and bump all of the ``Set a Collation...`` subsections up to the first heading level (thus making this article have a max heading depth of 2).

Reply:
Okay, changed the heading levels